### DISCUSSION TITLE
issue installing docker compose

### DISCUSSION THREAD

**Discussion by ArielAssaf** [advplyr/audiobookshelf#578] [Q&A] [answered]
Hi, I am new to docker so I appologize if this is obvious. 
I created the yaml file and ran it to get this error:

ERROR: Named volume "</mnt/Personal/Audiobooks/audiobooks>:/audiobooks:rw" is used in service "audiobookshelf" but no declaration was found in the volumes section.

Any idea what i did wrong?

**Comment by advplyr** [comment, marked as answer]
You must not have the "<" and ">" those are meant to highlight the paths you need to replace.

**Reply by ArielAssaf** [reply]
thank you!
